src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -79,6 +77,13 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
 
                     loggingBuilder.AddDefaultWebJobsFilters();
                     loggingBuilder.AddWebJobsSystem<WebHostSystemLoggerProvider>();
+
+                    if (!FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagDisableDiagnosticEventLogging))
```

Comment:
@soninaren -- these were only available from the ScriptHost before (I think) -- any objection to me also registering them at the WebHost level? It doesn't seem like there'd be any conflicts.

Reply:
No objections on my end.